src/course2/lesson5_exceptions/Main.java

```diff
@@ -1,6 +1,37 @@
 package course2.lesson5_exceptions;
+
 public class Main {
+    public static void main(String[] args) {
+        try {
+            hereWillBeTrouble("java_skypro_go", "D_1hWiKjjP_9", "D_1hWiKjjP_9");
+        } catch (WrongLoginException | WrongPasswordException e) {
+            System.err.println("Не верно указаны входные данные");
```

Comment:
```suggestion
            System.err.println("Не верно указаны входные данные: " + e.getMessage());
```
причину исключения лучше вывести, не просто же так она была передана через конструктор